src/eudplib/qgc/qgc.py

```diff
@@ -199,7 +211,7 @@ def QueueGameCommand_RemoveSelect(n, ptr_arr):  # noqa: N802
 
 
 @c.EUDFunc
-def QueueGameCommand_RightClick(xy):  # noqa: N802
+def QueueGameCommand_RightClick(xy: int):  # noqa: N802
```

Comment:
```suggestion
def QueueGameCommand_RightClick(xy: Dword):  # noqa: N802
```
Maybe it should be like this?

After checking the function calls to ``c.SetVariables``, it seems like it should be a ``Dword``.

Reply:
Yeah it looks good to me.